service/internal/proctelemetry/process_telemetry.go

```diff
@@ -127,6 +172,83 @@ func RegisterProcessMetrics(registry *metric.Registry, ballastSizeBytes uint64)
 	return nil
 }
 
+func (pm *processMetrics) createOtelMetrics() error {
+	var errors, err error
+
+	pm.otelProcessUptime, err = pm.meter.AsyncFloat64().Counter(
```

Comment:
I'm starting to think that for this one we can just have a simpler error just like the opencensus metrics
```
pm.otelProcessUptime, err = createOtelInstrument...
if err != nil {
	return err
}
err = registerCallback...
if err != nil {
	return err
}
```

we started this pattern of joining errors when the obsreport structs didn't return an error yet which is not the case here.

Reply:
Thanks for pointing this out! I see now that for the obsreport PRs the `handleError` function was simply logging a warning and not returning the individual errors. Changed back to the original pattern.